tardis/tests/integration_tests/report.py

```diff
@@ -67,14 +70,16 @@ def __init__(self, dokuwiki_details):
         else:
             self.dokuwiki_url = dokuwiki_details["url"]
 
+    def pytest_sessionstart(self, session):
```

Comment:
I found a method which could do the trick, I updated it in my subsequent commit.